coala_quickstart/green_mode/QuickstartBear.py

```diff
@@ -0,0 +1,31 @@
+from coalib.bears.LocalBear import LocalBear
+
+
+class QuickstartBear(LocalBear):
+    """
+    A bear to parse out the file_dict and return
+    values of settings that can take an infinite
+    set of values and can be determined easily
+    by looking at the file contents.
+    """
+
+    def run(self, filename, file):
+        """
+        :return:
+            A dict of setting name as key and the appropriate
+            value guessed by the bear.
+        """
+        max_line_length = 0
+        if file is not None:
+            for line in file:
+                length = len(line)
+                if length > max_line_length:
+                    max_line_length = length
+
+            return_val = dict()
+            return_val['max_line_length'] = max_line_length
+            return_val['max_lines_per_file'] = len(file)
+            return_val['min_lines_per_file'] = len(file)
+            return [return_val]
+        else:
```

Comment:
I think when you have a `if : return ..`, the `else` isnt necessary or desirable.

it would automatically return `None` if the `if` condition is not satisfied.

Btw, why you need to return `[None]`? Anyway, In this case, you could return it early if the file is `None`.
e.g.:
```
if not file:
    return [None]
```
Then you won't need do `if file:` condition.

Reply:
in fact that's the only line having partial coverage and I could use a little help on mocking the return value of `bear.run()` method over here

Reply:
>Btw, why you need to return [None]

okay that's because I didn't want to modify the code at `green_mode.run_quickstartbear` for a specific condition